components/02-elements/badge/_badge.scss

```diff
@@ -0,0 +1,20 @@
+.badge {
+    font-size: 12px;
+    padding: 5px 10px;
+    display: inline-flex;
+
```

Comment:
By default, the badge should have some kind of style too. Always every BEM block should be able to be used without any modificators.